Various fixes (#70)
epinio/docs: src/explanations/advanced.md

wiring is handled by what is called, an Ingress controller.

Most clusters will have an Ingress controller deployed by default. If that's not the case, [Traefik](https://doc.traefik.io/traefik/providers/kubernetes-ingress/) can be installed.
Other Ingress controllers should work too but Traefik is what is used in Epinio CI to test it.

Read the installation documentation on how to install Traefik: [Install ingress controller](../installation/installation.html#ingress-controller)

### Cert Manager

[Upstream documentation](https://cert-manager.io/docs/)

Cert Manager is a Kubernetes controller that generates and renews Certificates needed in order to
serve the various accessible endpoints over TLS (e.g. the Epinio API server).

Epinio supports various options when it comes to certificate issuers (let's encrypt, private CA, bring your own CA, self signed certs).
Cert Manager simplifies the way we handle the various different certificate issuers within Epinio.

You can read more about certificate issuers here: [certificate issuers documentation](../howtos/certificate_issuers.md)

## Epinio installed components

> The official way to install Epinio is the [Epinio helm chart](https://artifacthub.io/packages/helm/epinio/epinio).

Reviewer: ```suggestion
The official way to install Epinio is with the [Epinio helm chart](https://artifacthub.io/packages/helm/epinio/epinio).
```

> This helm chart installs Epinio and some additional components that are needed for Epinio to work.

Reviewer: ```suggestion
This helm chart installs Epinio and additional components, listed below, that are needed for Epinio to work.
```

The staging Job requires that image to be written to some container registry (See also [Detailed push process](../explanations/detailed-push-process.md)). 

By default the Epinio installation deploys a container registry inside the Kubernetes cluster to make the process easy and fast.

Epinio comes with two consumers of this registry:

1. Staging job - pushing the images
2. Kubernetes - pulling the images when a Deployment is created for the Application

Ideally all consumers will communicate with the registry over TLS to keep the communication encrypted.
Epinio controls the staging job and ensures that whatever CA is used to sign the registry certificate is trusted by it. Achieving the same for Kubernetes however requires configuration that cannot happen from within the cluster, therefore Epinio has no way to ensure that. Theoretically there are 3 options:

1. Let the Epinio user manually configure Kubernetes to trust the CA

> 2. Use a well-known trusted CA, so that no configuration is needed

Reviewer: ```suggestion
2. Use a well-known trusted CA, so there's no configuration needed
```
